tests/integration/olp-cpp-sdk-dataservice-write/VersionedLayerClientTest.cpp

```diff
@@ -559,4 +562,90 @@ TEST_F(VersionedLayerClientTest, CompleteBatchCancel) {
   }
 }
 
+TEST_F(VersionedLayerClientTest, CancelBatch) {
```

Comment:
I would like to see more tests, e.g. unit tests, CancelBatch with callback, cancel on deleting the client.